bofire/data_models/strategies/predictives/multi_fidelity.py

```diff
@@ -0,0 +1,20 @@
+from typing import Literal
+
+from pydantic import model_validator
+
+from bofire.data_models.strategies.predictives.sobo import SoboStrategy
+
+
+class MultiFidelityStrategy(SoboStrategy):
+    type: Literal["MultiFidelityStrategy"] = "MultiFidelityStrategy"
```

Comment:
`SoboStrategy` is currently allowing for a versatile set of acquisition functions. For this strategy only UCB is allowed or? 

Then we should overwrite here the acqf with:

```
acquisition_function: qUCB= Field(
        default_factory=lambda: qUCB()
    )
```

If you need UCB specific user settings (as discussed somewhere below), you can also add more parameters to the `qUCB` data model.

Reply:
The algorithm consists of first optimizing $x$ on the target fidelity, then optimizing the selection of fidelity, $m$. Since the first step only operates on a single fidelity, we can use any acquisition function that we choose (as noted in Folch et al 2023, sec 5.2).

The selection of $m$ then uses a variance-based acquisition function, which is contained entirely in `MultiFidelityStrategy`.

Reply:
Thank you very much for the clarification!